plugins/modules/ome_application_certificate.py

```diff
@@ -96,7 +97,7 @@
     ca_path: "/path/to/ca_cert.pem"
     command: "generate_csr"
     distinguished_name: "hostname.com"
-    subject_alternative_names: "hostname1.chassis.com, hostname2.chassis.com"
```

Comment:
can you once test with 

"hostname1.chassis.com, hostname2.chassis.com"

have space after comma. are we not supporting this pattern?

Reply:
yes,it will work